### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_MigrationAssessment_DevTool:There is no name defined for the more info link.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15739]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2;#AINotInScope;#DevTool

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and tab till view/select button and press enter.
7. Tab till more info link.
8. Open Devtool using ctrl+shift+I and hover over to the link.

### Actual:
There is no name defined for the more info link.

### Expected:
There should be a name more info defined for the link.

### User Impact:
Screen Reader users will not be able to know the purpose of link.

**Comment by goyal-anjali** [comment]
Hi @Shivamd1 
Could you please specify which 'More Info link' requires a name?
The link already has a name value but the label doesn't. Does label require a name?

**Comment by alanrenmsft** [comment]
![image](https://user-images.githubusercontent.com/13777222/127549832-1b418c2f-4759-4813-a268-37804d97f6bb.png)
@Shivamd1 please reopen if this is not what you were referring to.

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro 
Version:- 1.42.0-insider(user-setup)

![15739_issue fixed](https://user-images.githubusercontent.com/93374043/218377231-ac534d65-5d85-48a1-91c6-984bd26b3814.png)

**GithubTags**:#Closed;
